ios/RNFingerprintjsPro.swift

```diff
@@ -15,6 +15,10 @@ class RNFingerprintjsPro: NSObject {
         super.init()
     }
 
+    @objc static func requiresMainQueueSetup() -> Bool {
+        return false
+    }
```

Comment:
In Swift, you can simplify such a one-line method definitions by omitting the `return` keyword. Also, I suggest to place the `@objc` attribute in a separate line for consistency with other method declarations.

```suggestion
    @objc
    static func requiresMainQueueSetup() -> Bool { false }
```